src/Default/default.ts

```diff
@@ -199,7 +199,7 @@ export class DefaultContainer extends WebContainerBase {
             if (permission === "denied") {
                 console.warn("Notifications not permitted");
             } else if (permission === "granted") {
-                new (<any>this.globalWindow).Notification(options.title, { body: options.message }); // tslint:disable-line
+                new (<any>this.globalWindow).Notification(title, options); // tslint:disable-line
```

Comment:
Flagged for unused expression (the return of the constructor isn't assigned to a var or used)